app/models/spree/flow_io_order_decorator.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
-# `:display_total` modifications to display total prices beside Spree default. Example: https://i.imgur.com/7v2ix2G.png
-module Spree # rubocop:disable Metrics/ModuleLength
+module Spree
   # Added flow specific methods to Spree::Order
-  Order.class_eval do
-    serialize :meta, ActiveRecord::Coders::JSON.new(symbolize_keys: true)
+  module FlowIoOrderDecorator
+    def self.included(base)
+      base.serialize :meta, ActiveRecord::Coders::JSON.new(symbolize_keys: true)
 
-    store_accessor :meta, :flow_data
+      base.store_accessor :meta, :flow_data
 
-    before_save :sync_to_flow_io
-    after_touch :sync_to_flow_io
+      base.before_save :sync_to_flow_io
```

Comment:
@texpert Aren't we synchronizing the order twice when `.save` method is called?

Reply:
Yeah, this should be definitely refactored - I have created an issue for this - https://github.com/mejuri-inc/flowcommerce_spree/issues/23